verilog/analysis/checkers/struct_union_name_style_rule.cc

```diff
@@ -59,16 +62,16 @@ std::string StructUnionNameStyleRule::GetDescription(
       GetStyleGuideCitation(kTopic), ".");
```

Comment:
We need to add a description about the configurable options here, so that they are shown to the user and show up in the auto generated documentation; here is an [example how a parameter output](https://google.github.io/verible/lint.html#parameter-name-style) looks like.